backstop/tests/table.html

```diff
@@ -1867,1876 +1242,4740 @@ <h2 id="regular">Zebra striping</h2>
+              class="iui-button iui-borderless iui-small iui-active"
+              aria-label="Page 1"
+              type="button"
+            >
+              <span class="iui-label">1</span>
+            </button>
+
+            <button
+              class="iui-button iui-borderless iui-small"
+              aria-label="Page 2"
+              type="button"
+            >
+              <span class="iui-label">2</span>
+            </button>
+
+            <button
+              class="iui-button iui-borderless iui-small"
+              aria-label="Page 3"
+              type="button"
+            >
+              <span class="iui-label">3</span>
+            </button>
+
+            <button
+              class="iui-button iui-borderless iui-small"
+              aria-label="Page 4"
+              type="button"
+            >
+              <span class="iui-label">4</span>
+            </button>
+
+            <button
+              class="iui-button iui-borderless iui-small"
+              aria-label="Page 5"
+              type="button"
+            >
+              <span class="iui-label">5</span>
+            </button>
+
+            <button
+              class="iui-button iui-borderless iui-small"
+              aria-label="Page 6"
+              type="button"
+            >
+              <span class="iui-label">6</span>
+            </button>
+
+            <button
+              class="iui-button iui-borderless iui-small"
+              aria-label="Page 7"
+              type="button"
+            >
+              <span class="iui-label">7</span>
+            </button>
+
+            <span class="iui-ellipsis iui-small">&mldr;</span>
+
+            <div class="iui-progress-indicator-radial iui-determinate iui-small">
+              <svg
+                class="iui-radial"
+                viewBox="0 0 40 40"
+                aria-hidden="true"
+              >
+                <circle
+                  class="iui-track"
+                  cx="20"
+                  cy="20"
+                  r="15.91549"
+                >
+                </circle>
+                <circle
+                  class="iui-fill"
+                  cx="20"
+                  cy="20"
+                  r="15.91549"
+                  style="stroke-dashoffset: 20;"
+                >
+                </circle>
+              </svg>
+            </div>
+          </span>
+
+          <button
+            class="iui-button iui-borderless iui-small"
+            aria-label="Next page"
+            type="button"
+          >
+            <!-- Use chevron-right.svg from @itwin/itwinui-icons package. -->
+            <svg
+              class="iui-icon"
+              aria-hidden="true"
+              viewBox="0 0 16 16"
+            >
+              <path d="m4.7 0-1.4 1.4 6.6 6.6-6.6 6.6 1.4 1.4 8-8z" />
+            </svg>
+          </button>
+        </div>
+
+        <div class="iui-right">
+          <label class="iui-input-container">
```

Comment:
This select component does not fit within the extra-condensed paginator.  Will be addressed in #63.
![Screen Shot 2021-05-18 at 3 49 09 PM](https://user-images.githubusercontent.com/849817/118715093-1058b500-b7d8-11eb-8f1b-2ad09c0c93b6.png)

Reply:
Its fixed.